extensions/amp-inline-gallery/1.0/README.md

```diff
@@ -0,0 +1,319 @@
+---
+$category@: layout
+formats:
+  - websites
+teaser:
+  text: Displays multiple similar pieces of content along a horizontal axis, with optional pagination dots and thumbnails.
+experimental: true
+bento: true
+---
+
+# Bento Inline Gallery
+
+## Usage
+
+The Bento Inline Gallery component uses a Bento Carousel component to display slides, with optional pagination dots and thumbnails. Both components must be properly installed for the environment (Web Component vs Preact). Typical usage might look like:
+
+[example preview="inline" playground="true" imports="bento-inline-gallery:1.0,bento-base-carousel:1.0"]
+
+```html
+<bento-inline-gallery>
+  <bento-inline-gallery-thumbnails aspect-ration="1.5" loop></bento-inline-gallery-thumbnails>
+  <bento-base-carousel snap-align="center" visible-count="1.2" loop>
+    <img src="server.com/static/inline-examples/images/image1.jpg">
+    <img src="server.com/static/inline-examples/images/image2.jpg">
+    <img src="server.com/static/inline-examples/images/image3.jpg">
+  </bento-base-carousel>
+  <bento-inline-gallery-pagination inset></bento-inline-gallery-pagination>
+</bento-inline-gallery>
+```
+
+[/example]
+
+### Web Component
+
+You must include each Bento component's required CSS library to guarantee proper loading and before adding custom styles. Or use the light-weight pre-upgrade styles available inline. See [Layout and style](#layout-and-style).
+
+The examples below demonstrate use of the `<bento-inline-gallery>` web component.
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-inline-gallery
+```
+
+```javascript
+import '@ampproject/bento-inline-gallery';
+```
+
+[/example]
+
+#### Example: Include via `<script>`
+
+The example below contains an `bento-inline-gallery` consisting of three slides with thumbnails and a pagination indicator.
+
+[example preview="top-frame" playground="false"]
+
+```html
+<head>
+  <script src="https://cdn.ampproject.org/custom-elements-polyfill.js"></script>
+  <script async src="https://cdn.ampproject.org/v0/bento-inline-gallery-1.0.js"></script>
+</head>
+<body>
+
+  <bento-inline-gallery id="inline-gallery">
+    <bento-inline-gallery-thumbnails aspect-ration="1.5" loop></bento-inline-gallery-thumbnails>
+    <bento-base-carousel snap-align="center" visible-count="1.2" loop>
+      <img src="server.com/static/inline-examples/images/image1.jpg">
+      <img src="server.com/static/inline-examples/images/image2.jpg">
+      <img src="server.com/static/inline-examples/images/image3.jpg">
+    </bento-base-carousel>
+    <bento-inline-gallery-pagination inset></bento-inline-gallery-pagination>
+  </bento-inline-gallery>
+
+</body>
+```
+
+[/example]
+
+#### Interactivity and API usage
```

Comment:
I think this entire section can be removed given there are no actions or events specific to `bento-inline-gallery`, and everything relevant is documented via `bento-base-carousel`

Reply:
removed